app/controllers/application_controller.rb

```diff
@@ -1,9 +1,18 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::Base
+  helper_method :current_user2
```

Comment:
@SemyzSerhiy let's rename this to `current_user` method. We do not need to the duplicated method. If you have any conflicts with other `current_user` method - feel free to remove previous version of this method